randovania/data/json_data/prime1.txt

```diff
@@ -4718,7 +4764,9 @@ Asset id: 413884678
   > Dock to Gathering Hall Access
       Trivial
   > Event - Open gate
-      Scan Visor
+      All of the following:
+          Scan Visor
+          Morph Ball Bomb or Morph Ball or Bomb Jump (Intermediate)
```

Comment:
This line seems weird. Since it's an OR this means that Bomb Jump can be done without Morph or Bombs.

Reply:
Should be fixed